### DISCUSSION TITLE
Python issues on openbsd

### DISCUSSION THREAD

**Discussion by Rhylx** [searx/searx#2298] [Installation issue: custom]
Hi,
I tried to install searx on openbsd but there seem there are some issues when running pip to get the web server, a binary linked to python gets compiled and it produces errors.
Is there anyone, who got it to work ?
Best,
Rhylx

**Comment by dalf** [comment]
Can you post the error log ?

**Comment by Rhylx** [comment]
Oh, sorry the issue isn't directly with searx but with WSGI which seems to use W|X and that security rules aren't compatible with openbsd... But it seems to be possible to write W^X rules to make it work out.

**Comment by Rhylx** [comment]
The development implementation seem to work though
